dev.yml

```diff
@@ -3,7 +3,7 @@ name: gozk-recipes
 up:
   - homebrew:
     - zookeeper
-  - go: 1.7.5
+  - go: 1.9.2
```

Comment:
How are we building the production binary?

I have noticed that we use [godeb](https://github.com/Shopify/cookbooks/blob/master/cookbooks/golang/recipes/godeb.rb) and it's completely broken. This is something we need to fix for ABS and shackle.

We could potentially write code that will not compile in production

Reply:
This is a library and doesn't produce a binary. We need Go for running the tests.